bats/tests/containers/factory-reset-containerd-shims.bats

```diff
@@ -0,0 +1,27 @@
+load '../helpers/load'
+
+local_setup_file() {
+    RD_USE_RAMDISK=false # interferes with deleting $PATH_APP_HOME
+}
+
+@test 'factory reset' {
+    delete_all_snapshots
+    rm -rf "$PATH_CONTAINERD_SHIMS"
+    factory_reset
+    assert_not_exists "$PATH_APP_HOME"
+}
+
+@test 'factory reset will not remove any shims' {
+    mkdir -p "$PATH_CONTAINERD_SHIMS"
+    touch "${PATH_CONTAINERD_SHIMS}/containerd-shim-bogus-v1"
```

Comment:
It would be good to remove this bogus shim (or better, all of `$PATH_CONTAINERD_SHIMS`) in a `local_teardown` to ensure it doesn't affect other tests.

Reply:
In general it is up to each test to ensure it starts from a known state, so does a factory reset on startup, and verifies everything else is as expected.

I made an exception for deployment profiles because they are not removed by a factory reset and also affect the local user after the tests are done.

I've added a `local_teardown_file` because the shims are also not deleted by a factory-reset, but I don't see how the bogus shim would cause any problems for the local user.